### PR TITLE
Add writeup on API doc updates

### PR DESCRIPTION
**Pull request by amyrlam**
Tagging Jen and locks in case they have any comment

### CODE REVIEW

File: source/2019-09-13-the-ember-times-issue-115.md [hunk 1]
```diff
@@ -61,13 +62,13 @@ The Apple Music web application [was launched](https://www.theverge.com/2019/9/5
...
 
-## [Section Title in Title Case 🐹](#section-url)
+## [API Doc Updates 📝](https://github.com/emberjs/ember.js/pulls?utf8=✓&q=is%3Apr+is%3Aclosed+%5BDOC%5D)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Thanks to [@locks](https://github.com/locks) and team for working on [Octanify-ing the API docs](https://github.com/emberjs/ember.js/pulls?utf8=✓&q=is%3Apr+is%3Aclosed+%5BDOC%5D)! These documentation upgrades are part of the [Quest Issue](https://github.com/emberjs/ember.js/issues/18250) to update the API docs to show Octane and Native Classes. [@jenweber](https://github.com/jenweber) has provided **detailed step-by-step instructions**, a YUIDoc styleguide, and some examples on the [Quest Issue](https://github.com/emberjs/ember.js/issues/18250), encouraging interested contributors to post in [#dev-ember-learning](https://discordapp.com/channels/480462759797063690/480777444203429888) on the [Ember Discord](https://api.emberjs.com/ember/release) with any questions.
```
Comment (jayjayjpg) [inline source/2019-09-13-the-ember-times-issue-115.md]: What do you think about the following?

```suggestion
Thanks to [@locks](https://github.com/locks) and team for working on [Octanify-ing the API docs](https://github.com/emberjs/ember.js/pulls?utf8=✓&q=is%3Apr+is%3Aclosed+%5BDOC%5D)! These documentation upgrades are part of the [Quest Issue](https://github.com/emberjs/ember.js/issues/18250) to update the API docs to show Octane and Native Classes. [@jenweber](https://github.com/jenweber) has provided **detailed step-by-step instructions**, a YUIDoc styleguide, and some examples on the [Quest Issue](https://github.com/emberjs/ember.js/issues/18250), encouraging interested contributors to post in [#dev-ember-learning](https://discordapp.com/channels/480462759797063690/480777444203429888) on the [Ember Discord](https://discordapp.com/invite/zT3asNS) with any questions. 
```

**Review by jayjayjpg — APPROVED** [review]
This reads great! I left one more small suggestion for an update of the link to the Ember Discord, but otherwise this seems already good to go :sparkles:

File: source/2019-09-13-the-ember-times-issue-115.md (hunk 1 above)
Comment (amyrlam) [reply source/2019-09-13-the-ember-times-issue-115.md]: @jessica-jordan I strangely couldn't tell what the diff was so committed it to see...and still can't tell? Did you mean to fix something and didn't?

Comment (amyrlam) [reply source/2019-09-13-the-ember-times-issue-115.md]: @jessica-jordan just realized you meant something about updating the link to the Ember Discord, will check the links upstream

**Pull request merged**